src/App.js

```diff
@@ -1,16 +1,54 @@
-import React from "react";
+import React, { useState } from "react";
 import "./App.css";
 import Header from "./components/Header";
-import Footer from "./components/Footer";
 import Main from "./components/Main";
+import Footer from "./components/Footer";
+
 function App() {
+  const [marker, allowMarker] = useState(false);
+  const [data, setData] = useState({
```

Comment:
I wonder if we can rename this state to be a little more descriptive. We will have quite a few state names in this app, but a name like mapCenter and setMapCenter might give more context.

Reply:
`data` is a really generic variable name. Jen's suggestion of a comment is a good one, but even better would be a name that's more descriptive. That way, when you see the variable in other places in the app, we won't have to refer back to this to understand what it represents.

Reply:
Good call. I changed this to `mapProperties`. Hope that helps with the clarification of what the variable is and contains at other component levels.